app/Decsys/Controllers/WebhooksController.cs

```diff
@@ -22,6 +24,23 @@ public class WebhooksController : ControllerBase
     {
         _webhooks = webhooks;
     }
+    
+    [HttpGet("generate-secret")]
+    [SwaggerOperation("Generate a webhook secret")]
+    [SwaggerResponse(200, "Webhook secret generated.")]
+    [SwaggerResponse(500, "Server failed to generate the secret.")]
+    public IActionResult GenerateSecret()
+    {
+        try
+        {
+            string secret =  Crypto.GenerateId(32,  CryptoRandom.OutputFormat.Hex);
+            return Ok(secret);
+        }
+        catch (Exception e)
```

Comment:
Why are you catching all exceptions? If we do generate a legit 500 error it will never show up, despite you adding the swagger response above.
It's better practice to catch specific (expected) exceptions. Otherwise, it's best to let things bubble up naturally.